apps/medusa-be/.env.template

```
@@ -1,6 +1,8 @@
 # NETWORK CONFIG FOR DOCKER
 REDIS_URL=redis://localhost:6379
 DATABASE_URL=postgres://postgres:postgres@localhost:5432/medusa
+# Legacy MySQL database for N1 migration (optional)
+LEGACY_DATABASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LEGACY_DATABASE_URL key should go before the REDIS_URL key

(UnorderedKey)